spec/index.bs

```diff
@@ -11026,58 +11129,73 @@ GPUQueue includes GPUObjectBase;
     ::
         Issues a write operation of the provided data into a {{GPUTexture}}.
 
+        Issue: This is an example of using `<div>`s for timelines.
+
         <div algorithm=GPUQueue.writeTexture>
-            **Called on:** {{GPUQueue}} |this|.
+            <div data-timeline=content>
+                **Called on:** {{GPUQueue}} |this|.
 
-            **Arguments:**
+                **Arguments:**
 
-            <pre class=argumentdef for="GPUQueue/writeTexture(destination, data, dataLayout, size)">
-                |destination|: The [=texture subresource=] and origin to write to.
-                |data|: Data to write into |destination|.
-                |dataLayout|: Layout of the content in |data|.
-                |size|: Extents of the content to write from |data| to |destination|.
-            </pre>
+                <pre class=argumentdef for="GPUQueue/writeTexture(destination, data, dataLayout, size)">
+                    |destination|: The [=texture subresource=] and origin to write to.
+                    |data|: Data to write into |destination|.
+                    |dataLayout|: Layout of the content in |data|.
+                    |size|: Extents of the content to write from |data| to |destination|.
+                </pre>
 
-            **Returns:** {{undefined}}
+                **Returns:** {{undefined}}
 
-            1. Let |dataBytes| be [=get a copy of the buffer source|a copy of the bytes held by the buffer source=] |data|.
-            1. Let |dataByteSize| be the number of bytes in |dataBytes|.
-            1. Let |texture| be |destination|.{{GPUImageCopyTexture/texture}}.
-            1. Let |contents| be the contents of the [=images=] seen by
-                viewing |dataBytes| with |dataLayout| and |size|.
+                1. Let |dataBytes| be [=get a copy of the buffer source|a copy of the bytes held by the buffer source=] |data|.
+                1. Let |texture| be |destination|.{{GPUImageCopyTexture/texture}}.
+                1. Issue the |device timeline steps| on the [=Device timeline=] of |this|.
+            </div>
 
-                Issue: Specify more formally.
-            1. Issue the following steps on the [=Device timeline=] of |this|:
+            The |device timeline steps| are:
```

Comment:
This whole algorithm is another example of a scenario where the algorithm progression is effectively linear across the different timelines, so breaking the sections out as psuedo-functions that are being called from the previous section feels unnecessary.